### PR TITLE
Add native acl reloading

### PR DESCRIPTION
**Pull request by gageorsburn**

### CODE REVIEW

File: app/utils/linux.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import * as fs from 'fs';
+import * as path from 'path';
```
Comment (tale) [inline app/utils/linux.ts:2]: Looks mostly good, I'll go through the PR soon to switch to the `fs/promises` API, don't worry about that.

File: package.json [hunk 2]
```diff
@@ -60,7 +60,7 @@
 		"@types/react-dom": "npm:types-react-dom@beta"
 	},
 	"engines": {
-		"node": ">=18.0.0"
+		"node": ">=19.0.0"
```
Comment (tale) [inline package.json:63]: What needs Node.js 19?

**Review by tale — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (gageorsburn) [reply package.json:63]: The crypto dependency

**Review by tale — APPROVED** [review]

**Comment by tale** [comment]
See f0e4868 😄

**Pull request closed without merging**